app/controllers/NotFoundController.scala

```diff
@@ -24,7 +24,8 @@ class NotFoundController @Inject()(
     * Create an Action to render an HTML page with a 404 message.
     */
   def index(currentPath: String): Action[AnyContent] = silhouette.UnsecuredAction.async {
-    println(s"User tried to access unavailable route '${currentPath}'")
-    Future.successful(NotFound(views.html.notFound()))
+      implicit request: RequestHeader =>
+          println(s"User tried to access unavailable route '${currentPath}'")
```

Comment:
Does println work in your config?

Reply:
yeah it does: 
![image](https://user-images.githubusercontent.com/16246147/34916891-48be0f9a-f93f-11e7-880c-6396d1fcce6d.png)